packages/swapper/src/manager/testData.ts

```diff
@@ -0,0 +1,106 @@
+import { ChainAdapterManager, ethereum } from '@shapeshiftoss/chain-adapters'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import Web3 from 'web3'
+
+import { SwapperName, TradeQuote } from '../api'
+import {
+  CowSwapper,
+  CowSwapperDeps,
+  ThorchainSwapper,
+  ThorchainSwapperDeps,
+  ZrxSwapper,
+} from '../swappers'
+import { ETH, FOX, WETH } from '../swappers/utils/test-data/assets'
+import { ZrxSwapperDeps } from '../swappers/zrx/types'
+
+const zrxEthereumSwapperDeps: ZrxSwapperDeps = {
+  web3: <Web3>{},
+  adapter: <ethereum.ChainAdapter>{
+    getChainId: () => KnownChainIds.EthereumMainnet,
+  },
+}
+
+export const zrxEthereumSwapper = new ZrxSwapper(zrxEthereumSwapperDeps)
```

Comment:
is this going to get included into the build, and hence instantiate swappers at runtime in web?

Reply:
Indeed. Fixed up in https://github.com/shapeshift/lib/pull/1123/commits/7d9fc7531fae1c585c60d37845774f144f0ecbdc.